gradle/libs.versions.toml

```diff
@@ -126,6 +127,8 @@
 
     accompanist-permissions = { group = "com.google.accompanist", name = "accompanist-permissions", version.ref = "accompanist-permissions"}
 
+    ktlint-gradle = { module = "org.jlleitschuh.gradle:ktlint-gradle", version.ref = "ktlint" }
```

Comment:
this is a library definition instead of a plugin definiton. The library definition is used in root `build.gradle` to give the root project the ktlint dependency.

While in the `allprojects` snippet we apply the plugin definition of ktlint